### PR TITLE
Conda

### PR DESCRIPTION
**Pull request by cjmartian**
This adds conda to the list of rules.

### CODE REVIEW

File: thefuck/rules/conda_mistype.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import re
+from thefuck.utils import replace_command
+
+
+def match(command):
+    '''
+    Match a mistyped command
+    '''
+    return 'conda' in command.script and "Did you mean 'conda" in command.output
```
Comment (scorphus) [inline thefuck/rules/conda_mistype.py:9]:
```suggestion
from thefuck.utils import for_app
from thefuck.utils import replace_command


@for_app("conda")
def match(command):
    """
    Match a mistyped command
    """
    return "Did you mean 'conda" in command.output
```

**Review by scorphus — APPROVED** [review]
Spot on, @cjmartian! 👍

Nit: for sake of consistency — and some additional features attached — how about using`@for_app("coda")`? I also suggested consistent quoting ☺️

edit: *Sorry for the late review*

**Comment by cjmartian** [comment]
@scorphus No worries, I know we are all busy. I made those changes, cheers!

File: thefuck/rules/conda_mistype.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import re
+from thefuck.utils import replace_command, for_app
+
+
+@for_app("conda")
+def match(command):
+    """
+    Match a mistyped command
+    """
+    return "conda" in command.script and "Did you mean 'conda" in command.output
```
Comment (scorphus) [inline thefuck/rules/conda_mistype.py:10]: Once the function is anotated with `@for_app("conda")` there's no need to check if it is included in `command.script`.
```suggestion
    return "Did you mean 'conda" in command.output
```

**Review by scorphus — APPROVED** [review]
Thanks for applying the changes earlier, @cjmartian.

There's one last thing, check the the most recent comment regarding `for_app`. I'll go ahead and apply that suggestion :-)

**Comment by scorphus** [comment]
Thanks, @cjmartian 👍

**Comment by cjmartian** [comment]
Thanks for that last commit, I see what you mean now. Thanks for merging @scorphus

**Pull request merged**